webserver/sf-perftest-metrics/app.yaml

```diff
@@ -1,32 +1,21 @@
-apiVersion: apps.redskyops.dev/v1alpha1
+apiVersion: apps.optimize.stormforge.io/v1alpha1
```

Comment:
Do we want to keep this around or replace with the generated experiment?

Reply:
This and other SF ones will need to connect to stormforger app and get a valid token in the experiment generation process. Hence, leaving them alone.